src/index.ts

```diff
@@ -20,22 +20,68 @@ import mapperReader from './mapperReader.js';
 import { MapperServer } from './mapperServer.js';
 import { BidiServerRunner } from './bidiServerRunner.js';
 import { IServer } from './iServer.js';
+import argparse from 'argparse';
+
+function parseArguments() {
+  var parser = new argparse.ArgumentParser({
+    add_help: true,
+    exit_on_error: false,
+  });
+
+  parser.add_argument('-p', '--port', {
+    help: 'Port, BiDi server should listen to. Default is 8080.',
+    default: 8080,
+  });
+
+  parser.add_argument('-hl', '--headless', {
```

Comment:
Can `type: 'boolean'` be added here to avoid the string&rarr;boolean conversion later?

Reply:
`argparse` supports only `int`, `float` and `str`: https://github.com/nodeca/argparse/blob/master/doc/port_difference.md